src/components/LHNOptionsList/OptionRowLHNData.js

```diff
@@ -86,6 +87,8 @@ function OptionRowLHNData({
     const policy = lodashGet(policies, [`${ONYXKEYS.COLLECTION.POLICY}${fullReport.policyID}`], '');
 
     const parentReportAction = parentReportActions[fullReport.parentReportActionID];
+    const transactionID = lodashGet(parentReportAction, ['originalMessage', 'IOUTransactionID'], '');
+    const transaction = lodashGet(transactions, [`${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`], '');
```

Comment:
Perhaps also for the policies as well?

Reply:
I don't think this is possible, because there's no way to access the `parentReportAction` by key. And this `parentReportAction` is necessary to access the transaction. So we can't do this in `withOnyx`.
Or am I missing something? 🤔

Reply:
I don't think so, because in the selector we don't have access to the props, so we couldn't use the `fullReport` to access the correct report action

Reply:
Yap, I think I can do that, but I can't make the same refactor to parentReportAction.
Well, I can, but I need to make a `withReportActions` HOC. I've developed this and it seems to work fine. Do you think this optimization is worth it?
Note: the optimization could also be applied to the policies prop.